packages/perseus/src/widgets/interactive-graphs/graphs/components/angle-indicators.tsx

```diff
@@ -144,24 +145,44 @@ interface AngleProps {
     range: [Interval, Interval];
 }
 
+export function getWholeAngleMeasure(coords: Vector2[], vertex: vec.Vector2) {
+    const startAngle = getAngleFromVertex(coords[0], vertex);
+    const endAngle = getAngleFromVertex(coords[1], vertex);
+    const angle = (startAngle + 360 - endAngle) % 360;
+
+    // return only whole angles
+    return parseFloat(angle.toFixed(0));
+}
+
+export function getClockwiseCoords(
+    coords: Vector2[],
+    vertex: vec.Vector2,
+    allowReflexAngles: boolean = false,
+) {
+    // Check if the points are clockwise or not, depending on whether we allow reflex angles
+    const areClockwise = clockwise([...coords, vertex]);
+    const shouldReverseCoords = areClockwise && !allowReflexAngles;
+
+    // Reverse the coordinates accordingly to ensure the angle is calculated correctly
+    return shouldReverseCoords ? coords : coords.reverse();
+}
```

Comment:
Thanks for refactoring this code!

It seems like this duplicates some of the logic from `getClockwiseAngle` in `interactive-graphs/math/angles.ts`. Could we use that function to calculate the angle measure here? `getClockwiseAngle` is used for scoring, and it would be nice if we could use the same function to calculate the angle label.

Reply:
oh yes, you're right, it looks like it combines both the `getClockwiseCoords` and the `getWholeAngleMeasure` methods into one! let me test it out

Reply:
ok, so I tested this out and using the `getClockwiseAngle` function introduces a regression in the graph 😢
![image](https://github.com/user-attachments/assets/7fefd2d6-8d74-492a-91be-ba5332b08f01)
The very small difference between how we determine whether the coordinates should be reversed is the culprit: 
In `getClockwiseAngle`
`const shouldReverseCoords = !areClockwise || allowReflexAngles;`
In `getWholeAngleMeasure` 
`const shouldReverseCoords = areClockwise && !allowReflexAngles;`